MapboxVision/VideoStream/VisionManagerDelegate.swift

```diff
@@ -63,6 +63,11 @@ public protocol VisionManagerDelegate: class {
      Tells the delegate that authorization status has changed.
     */
     func visionManager(_ visionManager: VisionManager, didAuthorizationStatusChanged status: AuthorizationStatus) -> Void
+    
+    /**
+     Tells the delegte that country which is used in the VisionSDK changed.
```

Comment:
What do you think about keeping `visionManagerDidFinishUpdate` method as a last delegate method? It would give a visual hint that this method is called last.